.gitignore

```diff
@@ -1,2 +1,7 @@
 /node_modules
 /index.js
+/dist
```

Comment:
```suggestion

```

Reply:
Likewise with built files - as per above.

Reply:
Oops - nevermind - this was from when I changed it to output to the root 😅
Have removed.